evap/results/exporters.py

```diff
@@ -195,7 +210,9 @@ def write_headings_and_evaluation_info(
         # One more cell is needed for the question column
         self.write_empty_row_with_styles(["default"] + ["border_left_right"] * len(evaluations_with_results))
 
-    def write_overall_results(self, evaluations_with_results, course_results_exist):
+    def write_overall_results(
+        self, evaluations_with_results: list[tuple[Evaluation, None]], course_results_exist: bool
```

Comment:
Does `None` allow us here to pass something in that is not `None` there? I think it should be `Any` if we want to express that we don't care?

Reply:
You resolved the discussion item but you didn't respond or fix the code.

Surely, the API here isn't that users have to pass in a list in the form of `[(evaluation1, None), (evaluation2, None), ...]`, but instead, it seems to me that you simply don't care about the second tuple element in each tuple. In that case, it seems to me that `None` is a wrong annotation and `Any` should be used here instead.